### PR TITLE
Add Security Policy

### PR DESCRIPTION
**Pull request by devlead**

### CODE REVIEW

**Review by gep13 — APPROVED** [review]
LGTM

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+| <= 0.38.4 | :x:                |
+
+## Reporting Security Issues
+
+⚠ **Please do not report security vulnerabilities through public GitHub issues.** ⚠
+
+Instead, either encrypted through Keybase chat
+  - [keybase.io/cakebuild](https://keybase.io/cakebuild)
+
+Or report them to using NuGet.org *Contact owners*
```
Comment (augustoproiete) [inline SECURITY.md:18]: "Or report them ~**to**~ using NuGet.org *Contact owners*"

Tiny suggestion:
- "Or report them using NuGet.org *Contact owners*"

or
- "Or report them to **us** using NuGet.org *Contact owners*"

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  - [Cake.Tool](https://www.nuget.org/packages/Cake.Tool)
+  - [Cake.Frosting](https://www.nuget.org/packages/Cake.Frosting)
+  - [Cake.CoreCLR](https://www.nuget.org/packages/Cake.CoreCLR)
+  - [Cake.Core](https://www.nuget.org/packages/Cake.Core)
+  - [Cake.Common](https://www.nuget.org/packages/Cake.Common)
+  - [Cake.Testing](https://www.nuget.org/packages/Cake.Testing)
+  - [Cake.NuGet](https://www.nuget.org/packages/Cake.NuGet)
+  - [Cake.Cli](https://www.nuget.org/packages/Cake.Cli)
+
+If you prefer to submit without logging in, send email to [security@cakebuild.net](mailto:security@cakebuild.net).
```
Comment (augustoproiete) [inline SECURITY.md:29]: Tiny suggestion: "... send **an** email to ..."

**Review by augustoproiete — APPROVED** [review]
LGTM. Added a couple of non-blocking suggestions i.e. Good to merge as-is

File: SECURITY.md (hunk 1 above)
Comment (devlead) [reply SECURITY.md:18]: Good feedback, all adjust to the latter 👍

File: SECURITY.md (hunk 2 above)
Comment (devlead) [reply SECURITY.md:29]: Sure I'll adjust.

**Review by nils-a — APPROVED** [review]
LGTM

**Pull request merged**